Code/Tools/ProjectManager/Source/PythonBindings.h

```diff
@@ -84,6 +84,7 @@ namespace O3DE::ProjectManager
 
         // Templates
         AZ::Outcome<QVector<ProjectTemplateInfo>> GetProjectTemplates() override;
+        AZ::Outcome<QVector<ProjectTemplateInfo>> GetProjectTemplatesForRepo(const QString& repoUri) override;
         AZ::Outcome<QVector<ProjectTemplateInfo>> GetProjectTemplatesForAllRepos() override;
         AZ::Outcome<QVector<TemplateInfo>> GetGemTemplates() override;
 
```

Comment:
I think most of the Get Repo methods in this file can be const

Reply:
Added const to the functions touched by this review (and functions that cascaded to) to keep changes to a minimum we should fix all the const correctness issues in another review/in development branch